packages/app-admin/src/plugins/Menu/community.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { AdminMenuCommunityPlugin } from "@webiny/app-admin/types";
+import { ListItem, ListItemGraphic } from "@webiny/ui/List";
+import { Icon } from "@webiny/ui/Icon";
+import { ReactComponent as CommunityIcon } from "@webiny/app-admin/assets/icons/icon-community.svg";
+
+import { i18n } from "@webiny/app/i18n";
+const t = i18n.ns("app-admin/navigation");
+
+const plugin: AdminMenuCommunityPlugin = {
```

Comment:
@BhuwanChandra I think instead of creating a plugin type for each list item. We should create some plugins with type say `"admin-drawer-footer-menu"`. For example,
```js
{
    type: "admin-drawer-footer-menu",
    name: "admin-drawer-footer-menu-source",
    render() {
        // Some custom component
        return <MenuItemSource />;
    }
}
```
And then render all `"admin-drawer-footer-menu"` plugins inside `packages/app-admin/src/plugins/Menu/Navigation/index.tsx:67`